src-java/rule-manager/rule-manager-implementation/src/main/java/org/openkilda/rulemanager/utils/RoutingMetadata.java

```diff
@@ -84,8 +88,25 @@ public int getValue() {
         }
     }
 
+    public enum HaSubFlowType implements NumericEnumField {
+        HA_SUB_FLOW_1(0),
+        HA_SUB_FLOW_2(1);
+
+        private final int value;
+
+        HaSubFlowType(int value) {
+            this.value = value;
+        }
+
+        @Override
+        public int getValue() {
+            return value;
+        }
+    }
+
     private static long makeValue(
-            Boolean lldpFlag, Boolean arpFlag, Boolean oneSwitchFlowFlag, Integer outerVlanId, Integer inputPort) {
+            Boolean lldpFlag, Boolean arpFlag, Boolean oneSwitchFlowFlag, Integer outerVlanId, Integer inputPort,
```

Comment:
I'd rather put parameters on the previous line and parameters after inputPort on the next one.